Xamarin.Forms.Core/Internals/InvalidationTrigger.cs

```diff
@@ -1,9 +1,9 @@
 using System;
 
-namespace Xamarin.Forms
+namespace Xamarin.Forms.Internals
 {
 	[Flags]
-	internal enum InvalidationTrigger
+	public enum InvalidationTrigger
```

Comment:
Yeah, I see that and that it's our convention. It's just that our story can't simply be, "If it's in Form's namespace then we have a higher backcompat bar". Not gonna loose any sleep over it but could be a point of confusion for customers.